web/app/js/components/Sidebar.jsx

```diff
@@ -102,7 +102,7 @@ export default class Sidebar extends React.Component {
               <ConduitLink to="/deployments">Deployments</ConduitLink>
             </Menu.Item>
             <Menu.Item className="sidebar-menu-item" key="/docs">
-              <ConduitLink to="https://conduit.io/docs/" absolute="true">Documentation</ConduitLink>
+              <ConduitLink to="https://conduit.io/docs/" absolute="true" targetBlank="true">Documentation</ConduitLink>
```

Comment:
As is, I get console errors when I run this branch. Because of the propType validations, these have to be `absolute={true} targetBlank={true}` to not be strings.

Though, I would also be fine with your original idea of removing absolute as an arg to ConduitLink and just making this one a normal link, since it's the only link in the codebase that uses `absolute`
```
              import { Link } from 'react-router-dom';

              <Link to="https://conduit.io/docs/" target="_blank">Documentation</Link>
```